README.md

```diff
@@ -8,6 +8,7 @@ by leveraging the modern web.
 ---
 
 [![Build Status](https://travis-ci.org/andela/apollo-ah-backend.svg?branch=staging)](https://travis-ci.org/andela/apollo-ah-backend)
+[![Coverage Status](https://coveralls.io/repos/github/andela/apollo-ah-backend/badge.svg?branch=staging)](https://coveralls.io/github/andela/apollo-ah-backend)
```

Comment:
so sorry I forgot to inform you that I have moved the badges to the top of the read me file.

Reply:
Thanks for informing me. I should be able to update my local repository accordingly.